src/store/ChartStore.js

```diff
@@ -146,7 +146,9 @@ class ChartStore {
         const _self = this;
 
         // Add custom injections to the CIQ
-        inject();
+        inject({
+            drawToolsStore: this.mainStore.drawTools,
```

Comment:
@morteza-binary  Designer asked us to update the drawing tool hover tooltip with the new naming design. and this is the only way we can update them.